### PR TITLE
Added doc page description markdown.

### PR DESCRIPTION
**Pull request by OGKevin**

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,734 @@
+# Introduction
+
+Welcome to bunq!
+
+- The bunq API is organised around REST. JSON will be returned in almost all responses from the API, including errors but excluding binary (image) files.
+- All calls made through bunq Doc are executed on a sandbox environment. No real money is used and no transactions to external bank accounts can be done.
+- Please configure your implementation to send it's API requests to https://sandbox.public.api.bunq.com/v1.
```
Comment (sandervdo) [inline DESCRIPTION.md]: `its`

File: DESCRIPTION.md [hunk 2]
```diff
@@ -0,0 +1,734 @@
...
+- There is an Android client available of our app that connects to the bunq Sandbox environment.
```
Comment (sandervdo) [inline DESCRIPTION.md]: `There is an version of the Android app that connect to ...`

File: DESCRIPTION.md [hunk 3]
```diff
@@ -0,0 +1,734 @@
...
+- If you have a developer key, please proceed to the Settings page and enter this key to enable execution of test calls.
+
+## Get Started
+
+1. Create a user account with your phone. Afterwards, you can use this account to create an API key from which you can make API calls. You can find API key management under 'Profile' -\> 'Security'.
```
Comment (sandervdo) [inline DESCRIPTION.md]: We have Tinker for this now?

File: DESCRIPTION.md [hunk 4]
```diff
@@ -0,0 +1,734 @@
...
+Once the speed of iteration slows down and more developers start using the API and its sandbox we will start versioning the API using the version number in the HTTP URLs (i.e. the '/v1' part of the path). We will inform you when this happens.
+
+# Authentication
+
+- We use encryption for all API calls. This means that all requests must use HTTPS. The HTTP standard calls will fail. You should also use SSL Certificate Pinning and Hostname Verification to ensure a secure connection with bunq.
+- In order to make API calls you need to register a device and open a session.
+- We use RSA Keys for signatures headers and encryption.
+- API calls must contain a valid authentication token in the headers.
+- The auto logout time that you've set for your user account is also effective for your sessions. If a request is made 30 minutes before a session expires, the session will automatically be extended.
+## Device Registration
```
Comment (sandervdo) [inline DESCRIPTION.md]: I believe the new strict Markdown requires enters around headers.
Can you doublecheck?

File: DESCRIPTION.md [hunk 5]
```diff
@@ -0,0 +1,734 @@
...
+### Using our SDKs
+1. In order to start making calls with the bunq API, you must first register your API key and device and create a session.
+2. In the SDKs, we group these actions and call it "creating an API context".
+3. You can find more information on our GitHub page
```
Comment (sandervdo) [inline DESCRIPTION.md]: Missing `.` at the end, and would make `GitHub page` clickable.

File: DESCRIPTION.md [hunk 6]
```diff
@@ -0,0 +1,734 @@
...
+
+Using a Wildcard API Key gives you the freedom to make API calls from any IP address after the POST device-server. You can switch to a Wildcard API Key by tapping on “Allow All IP Addresses” in your API Key menu inside the bunq app. Find out more at this link https://bunq.com/en/apikey-dynamic-ip.
+
+# Signing
+
+To avoid modification of API call data while in transit (i.e. man-in-the-middle attacks), we use a request/response signing system. The signature ensures that the data is coming from the party that has the correct private key.
+
+While this system is already implemented in our SDKs, you should always follow the guidelines on this page when using the bunq API to make sure you correctly sign your calls.
+
+The signatures are created using the SHA256 cryptographic hash function and included (encoded in base 64) in the `X-Bunq-Client-Signature` request header and `X-Bunq-Server-Signature` response header. The data to sign is the following:
```
Comment (sandervdo) [inline DESCRIPTION.md]: Is still reflecting the changes in the capitalisation of this header name?

File: DESCRIPTION.md (hunk 3 above)
Comment (OGKevin) [reply DESCRIPTION.md]: Nein. Tinker is for playtime not actually implementation. Then you should indeed go and manually create the key 😊

File: DESCRIPTION.md (hunk 6 above)
Comment (OGKevin) [reply DESCRIPTION.md]: Arg. Lets create follow up issue for this one.

Comment (OGKevin) [reply DESCRIPTION.md]: https://github.com/bunq/doc/issues/10

**Comment by OGKevin** [comment]
@sandervdo pushed

**Pull request merged**